delphi-frontend/src/test/java/au/com/integradev/delphi/symbol/scope/UnknownScopeImplTest.java

```diff
@@ -127,8 +128,13 @@ void testGetVariableDeclarations() {
   }
 
   private static NameOccurrence makeNameOccurrence() {
+    DelphiToken token = mock(DelphiToken.class);
```

Comment:
What's this change for?

Reply:
These tests were failing because the mocked node didn't have a token, which was previously fine but now is required to determine if it's an included `SymbolicNode` or not. So I mocked a token for it.